app/pages/dashboard/template.test.js

```diff
@@ -22,35 +47,58 @@ describe('organisations dashboard page', () => {
       expect(description.length).toEqual(1);
-      expect(description.text().trim()).toEqual(context.description);
+      expect(description.text().trim()).toEqual(content.description);
     });
   }));
 
   it('should render a add organisation button', createTestHarness(setup, (harness) => {
     harness.request(context, ($) => {
       const addOrgButton = $('[data-test-id="add-org-button"] a');
       expect(addOrgButton.length).toEqual(1);
-      expect(addOrgButton.text().trim()).toEqual(context.addOrgButtonText);
-      expect(addOrgButton.attr('href')).toEqual('#');
+      expect(addOrgButton.text().trim()).toEqual(content.addOrgButtonText);
+      expect(addOrgButton.attr('href')).toEqual(context.addOrgButtonHref);
     });
   }));
 
-  it('should render the table component', createTestHarness(setup, (harness) => {
+  it('should render the table component with data', createTestHarness(setup, (harness) => {
     harness.request(context, ($) => {
       const orgTable = $('[data-test-id="org-table"]');
+      const columnHeading1 = orgTable.find('[data-test-id="column-heading-0"]');
+      const columnHeading2 = orgTable.find('[data-test-id="column-heading-1"]');
+      const row1 = $('[data-test-id="table-row-0"]');
+      const row1OrgName = row1.find('a[data-test-id="org-name-org1"]');
+      const row1OdsCode = row1.find('div[data-test-id="ods-code-org1"]');
+      const row2 = $('[data-test-id="table-row-1"]');
+      const row2OrgName = row2.find('a[data-test-id="org-name-org2"]');
+      const row2OdsCode = row2.find('div[data-test-id="ods-code-org2"]');
+
       expect(orgTable.length).toEqual(1);
       expect(orgTable.find('[data-test-id="table"]').length).toEqual(1);
       expect(orgTable.find('[data-test-id="table-headings"]').length).toEqual(1);
-      expect(orgTable.find('[data-test-id="column-heading"]').length).toEqual(2);
-      // TODO: Add tests to check the organisations data comes through
+      expect(columnHeading1.length).toEqual(1);
+      expect(columnHeading1.text().trim()).toEqual(content.columnInfo[0].data);
+      expect(columnHeading2.length).toEqual(1);
+      expect(columnHeading2.text().trim()).toEqual(content.columnInfo[1].data);
+      expect(row1.length).toEqual(1);
+      expect(row1OrgName.length).toEqual(1);
+      expect(row1OrgName.text().trim()).toEqual(context.organisations[0][0].data);
+      expect(row1OrgName.attr('href')).toEqual(context.organisations[0][0].href);
+      expect(row1OdsCode.length).toEqual(1);
+      expect(row1OdsCode.text().trim()).toEqual(context.organisations[0][1].data);
+      expect(row2.length).toEqual(1);
+      expect(row2OrgName.length).toEqual(1);
+      expect(row2OrgName.text().trim()).toEqual(context.organisations[1][0].data);
+      expect(row2OrgName.attr('href')).toEqual(context.organisations[1][0].href);
+      expect(row2OdsCode.length).toEqual(1);
+      expect(row2OdsCode.text().trim()).toEqual(context.organisations[1][1].data);
```

Comment:
Over here can we go with hardcoding the expectations instead.

Reply:
how come?

Reply:
I have done it but personally I like to know where the data came from so you can tell if it is hard coded in the component, comes from the context or comes from the manifest